dylan-package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "objc-dylan",
+    "url": "https://github.com/dylan-foundry/objc-dylan",
+    "description": "An Objective C / Dylan bridge",
+    "version": "0.1.0",
+    "dependencies": [],
+    "dev-dependencies": [
+        "testworks"
+    ],
+    "keywords": [
+        "Mac OS X",
```

Comment:
This should just be `macOS` as that is the name of the OS for the last several years.